tests/Tests.AzureAppConfiguration/PushRefreshTests.cs

```diff
@@ -0,0 +1,313 @@
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken1"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken2"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = "SyncToken1"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = "SyncToken2"
+                                    }
+            };
+
+        List<PushNotification> invalidPushNotificationList = new List<PushNotification>
+            {
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken1;sn=001"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+
+									EventType = null,
+                                    SyncToken = "SyncToken2"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store1.resource.io/kv/searchQuery1"),
+
+									EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = null
+                                    },
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = null
+                                    },
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = null,
+                                    SyncToken = null
+                                    }
+            };
+
+		ConfigurationSetting FirstKeyValue => _kvCollection.First();
+
+        [Fact]
+        public void TryTryCreatePushNotification()
```

Comment:
Typo: `Try` is added twice to test name

Reply:
Also, we now have a public method with the same name so we shouldn't name the test like this. Can you rename this test accordingly?

Reply:
It was intentionally named `TryTryCreatePushNotification`. Since we are trying the `TryCreatePushNotification()` Method. I could rename it to `TryCreatePushNotificationPopulatesNotification()` or `CheckTryCreatePushNotification()` maybe?